### ISSUE TITLE
Controller tests fail on Rpi

### ISSUE THREAD

**Issue by martukas** [RespiraWorks/Ventilator#1024]
**Current behaviour**
None of the tests in `test.sh` manage to build and run on Raspberry Pi. Platformio installs and fetches dependencies/libraries, but the tests fail to link, it seems.

**Expected behaviour**

Tests should build and run.

**How do you know it is wrong**

It should be possible to build, test and deploy directly on the RPI hooked up to the nucleo.
This is likely how software updates will work for the final product and at this stage of prototyping it makes sens to be deploying like this as well.

**Code pointers**

```
==2138==ASan runtime does not come first in initial library list; you should either link runtime to your application or manually preload it with LD_PRELOAD.
```

**Comment by asmodai27** [comment]
Appears fixed since deployment scripts update: #1188 . We need to confirm this.

**Comment by martukas** [comment]
Yes, this works now.
